model/emoji.go

```diff
@@ -14,6 +14,2392 @@ const (
 	EMOJI_SORT_BY_NAME    = "name"
 )
 
+var systemEmojis = map[string]int{
```

Comment:
Do you have a script or something that you used to generate this? If you have something like that, could we integrate it into [the make-emojis script](https://github.com/mattermost/mattermost-webapp/blob/master/build/emoji/make-emojis) so that we can easily update this?

Alternatively, instead of having this hardcoded, we could have [emoji.json from the web app](https://github.com/mattermost/mattermost-webapp/blob/master/utils/emoji.json) copied into this repository and loaded at startup to create this map.

Either way, that would really help to make this easily to keep in sync between the server and web app

Reply:
I created the map from the emoji.json, so I guess the second option could be easier in this case. I agree that both files should be kept in sync.

Reply:
OK, so I started copying `emoji.json` to `model/` and got it working parsing it from there, now, how and when can we copy `utils/emoji.json` from the webapp to the server?